exporter/awsemfexporter/metric_translator.go

```diff
@@ -55,15 +57,29 @@ type rateState struct {
 	timestamp int64
 }
 
+// MetricInfo defines
+type MetricInfo struct {
+	Value  interface{}
+	Unit   string
+}
+
+// GroupedMetric defines
+type GroupedMetric struct {
+	Namespace    string
+	Timestamp    int64
+	Labels      map[string]string
+	Metrics     map[string]MetricInfo
```

Comment:
```suggestion
	Metrics     map[string]*MetricInfo
```
I would recommend using a pointer for memory-efficiency

Reply:
Should this be a list of `*MetricInfo`? What happens if we have 2 metrics with the same name but different value?
```suggestion
	Metrics     map[string][]*MetricInfo
```

Reply:
> Should this be a list of `*MetricInfo`? What happens if we have 2 metrics with the same name but different value?

I expected to have multiple groupedMetrics for such case. As there can be only one value for one field in EMF log event.

Reply:
Hmm, I see. What happens if we have the same namespace, timestamp, and label? This would not generate a different groupedMetric in the current batching implementation.